packages/flutter/test/material/text_field_test.dart

```diff
@@ -4632,9 +4630,54 @@ void main() {
     );
 
     await gesture.up();
-    await tester.pumpAndSettle();
+    await tester.pump();
     expect(find.byType(CupertinoButton), findsNWidgets(3));
-    debugDefaultTargetPlatformOverride = null;
+  });
+
+  testWidgets('tap on non-force-press-supported devices work (iOS)', (WidgetTester tester) async {
+    final TextEditingController controller = TextEditingController(
+      text: 'Atwater Peel Sherbrooke Bonaventure',
+    );
+    await tester.pumpWidget(
+      MaterialApp(
+        theme: ThemeData(platform: TargetPlatform.iOS),
+        home: Material(
+          child: TextField(
+            controller: controller,
+          ),
+        ),
+      ),
+    );
+
+    final Offset textfieldStart = tester.getTopLeft(find.byType(TextField));
+
+    const int pointerValue = 1;
+    final Offset offset = textfieldStart + const Offset(150.0, 5.0);
```

Comment:
Here and elsewhere: is this offset suppose to match a location in the controller's text?

Reply:
Yes. Do you prefer using `textOffsetToPosition`? I don't have a strong opinion though I felt textOffsetToPosition was a bit round-about-ish.

Reply:
It's no big deal, I was just wondering what we expected the x=150 to line up with.